test/test_jit.py

```diff
@@ -15490,92 +15491,6 @@ def fn(x):
     'lu'
 }
 
-def check_alias_annotation(method_name, args, kwargs):
```

Comment:
These functions were just moved to common_jit.py so test_ops.py could also utilize them